ballerina/Ballerina.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 org = "ballerina"
 name = "io"
-version = "1.6.1"
+version = "1.6.2"
```

Comment:
Let's bump to a minor version

Reply:
Fixed with [cd3bc8bacfcffc4ff18a7f404280d6148e6b8798](https://github.com/ballerina-platform/module-ballerina-io/pull/1300/commits/cd3bc8bacfcffc4ff18a7f404280d6148e6b8798)